CMakeLists.txt

```diff
@@ -1,59 +1,119 @@
 # Require at least version 2.8.12 of CMake
 cmake_minimum_required(VERSION 2.8.12)
 
+if(CMAKE_VERSION VERSION_EQUAL 3.0 OR CMAKE_VERSION VERSION_GREATER 3.0)
+    cmake_policy(SET CMP0042 NEW)
```

Comment:
See the following commit for the details of why it was needed:
https://github.com/ellismg/coreclr/commit/abb7b0ecfd7b7c589b2e9d775da615416343a19f